ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/OptimizeQueueService.java

```diff
@@ -850,5 +787,133 @@ private List<OptimizeTaskItem> addTask(TableOptimizeItem tableItem, List<BaseOpt
 
       return toExecuteTasks;
     }
+
+    /**
+     * A policy that determines how to schedule a table.
+     * Tables at the head of the list have higher optimizing priority
+     */
+    interface SchedulePolicy {
```

Comment:
The interface and the implements will be clear in structure if they are external classes.
Otherwise, the OptimizeQueueService class contains too much code.

Reply:
I don't understand what you mean, can you explain in detail

Reply:
> I don't understand what you mean, could you explain in detail

I mean the interface SchedulePolicy and implement defined as external classes rather than the inner class will be better?